MobileWallet/Screens/AppEntry/Splash/WalletCreationViewController.swift

```diff
@@ -214,23 +214,21 @@ class WalletCreationViewController: UIViewController {
             DispatchQueue.main.async {
                 Tracker.shared.track("/onboarding/enable_push_notif", "Onboarding - Enable Push Notifications")
 
-                let storyboard = UIStoryboard.init(name: "Main", bundle: nil)
-
-                guard
-                    let navController = storyboard.instantiateInitialViewController() as? UINavigationController,
-                    let homeViewController = navController.viewControllers.first as? HomeViewController,
-                    let window = UIApplication.shared.windows.first
-                else { return }
-
-                let overlayView = UIScreen.main.snapshotView(afterScreenUpdates: false)
-                homeViewController.view.addSubview(overlayView)
-                window.rootViewController = navController
-
-                UIView.animate(withDuration: 0.4, delay: 0, options: .transitionCrossDissolve, animations: {
-                    overlayView.alpha = 0
-                }, completion: { _ in
-                    overlayView.removeFromSuperview()
-                })
+                let nav = AlwaysPoppableNavigationController()
```

Comment:
Please avoid abbreviations, make sure the name is meaningful on its own. From Apple's [Swift API Design Guidelines](https://swift.org/documentation/api-design-guidelines/#general-conventions):

__**Avoid abbreviations.** Abbreviations, especially non-standard ones, are effectively terms-of-art, because understanding depends on correctly translating them into their non-abbreviated forms.

The intended meaning for any abbreviation you use should be easily found by a web search.__